invenio_rdm_records/services/schemas/files.py

```diff
@@ -19,11 +19,12 @@
 class FileSchema(Schema):
     """File schema."""
 
-    checksum = fields.String()
+    id = fields.String(attribute="file.id")
```

Comment:
This is a tiny bit "internal", but could be helpful for caching and other purposes.